bgpv2: Status reporting document (#36134)
cilium-team/cilium: Documentation/network/bgp-control-plane/bgp-control-plane-operation.rst

            Applied Keepalive Seconds:  30
    Events:                             <none>

Disabling CRD Status Report
===========================

CRD status reporting is useful for troubleshooting. Thus, it is recommended to

> enable it in general. However for the large cluster which has a lot of nodes or

Reviewer: ```suggestion
enable it in general. However, for large clusters which have a lot of nodes or
```

Author: Thanks, fixed!

> CRD status reporting is useful for troubleshooting. Thus, it is recommended to
> enable it in general. However, for large clusters which have a lot of nodes or
> BGP policies, it may make a significant API server load. If you wish to disable
> status reporting, you can set ``bgpControlPlane.statusReport.enabled`` Helm
> value to ``false``. It disables the status reporting and clears the currently
> reported status.

Reviewer 2: Tweaks for active voice and clarity
```suggestion
CRD status reporting is useful for troubleshooting, making it useful to
enable in general. However, for large clusters with a lot of nodes or
BGP policies, CRD status reporting may add a significant API server load. 
To disable status reporting, set the ``bgpControlPlane.statusReport.enabled`` Helm
value to ``false``. Doing so disables status reporting and clears the currently
reported status.
```

Author: Applied, thanks!
